Jenkinsfile

```diff
@@ -30,13 +30,22 @@ timestamps {
       }
       params = [
         string(name: 'svn_revision', value: 'main'),
-        string(name: 'branch', value: 'main'),
```

Comment:
John's commit removes `svn_revision` instead of branch: https://github.com/snowflakedb/snowflake-jdbc/pull/1190/files

Reply:
yea this shud be `svn_revision`

Reply:
branch keeps to be `main`